.travis.yml

```diff
@@ -4,12 +4,11 @@ language: python
 
 matrix:
   include:
-    - {python: 3.5, env: TOX_ENV=py35}
-    - {python: 3.6, env: TOX_ENV=py36}
-    # Need sudo here to actually get xenial.
-    # See https://github.com/travis-ci/travis-ci/issues/9815
-    - {python: 3.7, env: TOX_ENV=py37, dist: xenial, sudo: true}
-    - {python: 3.8, env: TOX_ENV=py38, dist: xenial, sudo: true}
+    - {python: 3.7, env: TOX_ENV=py37}
+    - {python: 3.8, env: TOX_ENV=py38}
+    - {python: 3.9, env: TOX_ENV=py39}
+    - {python: 3.10, env: TOX_ENV=py310}
+    - {python: 3.11, env: TOX_ENV=py311}
```

Comment:
Is Travis CI still running for this project? You may need to set up GitHub workflows first.

Reply:
Yep, I realized that after creating this commit.  My overall plan is https://github.com/thriftrw/thriftrw-python/issues/166#issuecomment-1426644007 which does include setting up Github Actions.

Reply:
Github Actions is set up in #170